paket.dependencies

```diff
@@ -176,6 +176,11 @@ group netcore
     nuget Octokit
     nuget Newtonsoft.Json
     nuget Paket.Core // prerelease
+    nuget NuGet.Protocol < 5 // support net462
```

Comment:
Maybe change this to `nuget NuGet.Protocol ~> 4 strategy: min`? You could omit the others then.

Reply:
yes true, but I don't want min I want latest 4.X - but in this scenario it might be equivalent ;)
Good idea